### DISCUSSION TITLE
SSR styles rendering

### DISCUSSION THREAD

**Discussion by Arbust908** [frandiox/vite-ssr#149] [Q&A] [answered]
Hi there I recently migrated my Vitesse-based web project to use vite-ssr instead of the SSG. And noticed that styles seem to be loading after the HTML is rendered. And  it's taking its time.
So is it a dev environment only thing? Is there a setting I'm not seeing? I don't see critters (the library SSG uses to inject styles) anywhere. 

I could provide a project link if needed. (Not on my PC at the time)

**Comment by frandiox** [comment, marked as answer]
Yes, it should happen only in development. It is a known issue in Vite: https://github.com/vitejs/vite/issues/2282
There are a few workarounds listed in that thread, in case you want to try some.
